test/test_jit.py

```diff
@@ -69,6 +69,43 @@ def f(x, y):
         torch._C._jit_pass_lint(trace)
         self.assertExpected(str(trace))
 
+    def test_matmul_native(self):
```

Comment:
Can you comments explaining what these tests are checking (so we remember in the future)?